storages/backends/gs.py

```diff
@@ -1,3 +1,14 @@
+import warnings
+warnings.warn("DEPRECATION NOTICE: This backend is deprecated in favour of the "
```

Comment:
The syntax is fine. `warn` takes a second argument of the type to issue which in this case should be `DeprecationWarning` or `PendingDeprecationWarning`.